tools/number-converter/src/NumberConverter.vue

```diff
@@ -110,16 +132,62 @@ export default {
 				this.errorMessage = err.message;
 			}
 		},
-		convertToBin(number) {
-			return number.toString(2);
+		convertToDec(number, fromBase) {
```

Comment:
Lot's of repetition in `convertToDec`, `convertToBin`, etc... perhaps could use decimal as a 'middle' step? I.e. pseudocode for `onConvertClick`:

```
onConvertClick() {
    const value = convertFromBase(this.inputNumber, this.fromBase);
    const converted = convertToBase(value, this.toBase);
}
```

where `convertFromBase` and `convertToBase` are focused on converting numbers from base 10 to and from other bases respectively.

Reply:
I did this slightly differently, but with the same result of simplifying the code!